samples/drivers/watchdog/Makefile

```diff
@@ -1,4 +1,12 @@
 BOARD ?= quark_d2000_crb
-CONF_FILE = prj.conf
+
+ifeq ($(BOARD),96b_carbon)
+CONF_FILE ?= prj_$(BOARD).conf
+else
+ifeq ($(BOARD),nucleo_f334r8)
+CONF_FILE ?= prj_$(BOARD).conf
```

Comment:
@apodogrocki: Ok, that's strange. What about doing this configuration in iwdg_main.c?
This approach has been used for several sample/basic samples and seems to work.
It's less nice but open the possibility of test to whole stm32 family